apps/wallet-dashboard/lib/utils/migration/groupMigrationObjects.ts

```diff
@@ -25,6 +25,7 @@ export async function groupMigrationObjectsByUnlockCondition(
     client: IotaClient,
     currentAddress: string = '',
     groupByTimelockUC: boolean = false, // If true, group by timelock unlock condition, else group by expiration unlock condition
+    currentEpochStartMs: number,
```

Comment:
It is optional, because in case you group by timelockUC, you don't need it
```suggestion
    currentEpochStartMs?: number,
```